LayoutTests/pointer-lock/mouse-event-delivery.html

```diff
@@ -57,11 +58,14 @@
             targetdiv1.onwheel = eventExpectedThenNextStep;
             targetdiv2.onwheel = eventNotExpected;
             document.body.onwheel = eventExpected;
-            if (window.eventSender) {
-                window.eventSender.mouseMoveTo(20,20);
-                window.eventSender.leapForward(1000);
-                window.eventSender.mouseScrollBy(0,10);
-            }
+            setTimeout(async () => {
+                await UIHelper.ensurePresentationUpdate();
```

Comment:
I take it you found the `setTimeout(..., 0)` really is necessary here, and you can't just await the presentation update immediately?